config/application.rb

```diff
@@ -159,6 +163,13 @@ class Application < Rails::Application
       Vmdb::Loggers.apply_config(::Settings.log)
     end
 
+    initializer :eager_load_all_the_things, :after => :load_config_initializers do
```

Comment:
LOL `:eager_load_**all_the_things**`

Reply:
Let me pull this and the other debug logging into a conditional, default disabled.

Reply:
ok, done